core/DataTable/Filter/ColumnCallbackAddMetadata.php

```diff
@@ -45,10 +45,6 @@ public function __construct($table, $columnsToRead, $metadataToAdd, $functionToA
     {
         parent::__construct($table);
 
-        if (!is_array($columnsToRead)) {
-            $columnsToRead = array($columnsToRead);
-        }
-
         $this->columnsToRead      = $columnsToRead;
```

Comment:
I can remove it if you want, I don't care about it. It's only a bit of duplicated code and right next to each other not across files or functions or whatever. In general duplicated code is not good, yes, that's why I added that comment initially but sometimes it's ok. Anyway, I will just remove it